Cargo.toml

```diff
@@ -3,15 +3,15 @@ name = "sigchat"
 version = "0.1.0"
 authors = ["john <john@nhoj.info>"]
 edition = "2018"
-description = "Matrix chat"
+description = "sigchat"
 
 # Dependency versions enforced by Cargo.lock.
 [dependencies]
 log = "0.4.14"
 num-derive = { version = "0.3.3", default-features = false}
 num-traits = { version = "0.2.14", default-features = false}
-xous = "0.9.52"
-xous-ipc = "0.9.52"
+xous = "0.9.56"
```

Comment:
current latest version s 0.9.62 but i think none of the changes impact sigchat -- all stuff around swap memory, so this is fine. just noting that.